modules/system-target.nix

```diff
@@ -9,6 +9,7 @@ let
   config_types = {
     aarch64-linux = "aarch64-unknown-linux-gnu";
     armv7l-linux = "armv7l-unknown-linux-gnueabihf";
+    armv7a-linux = "armv7a-unknown-linux-gnueabihf";
```

Comment:
With the previous comment, this would be dropped.

Reply:
dropped